pandas/core/computation/expr.py

```diff
@@ -38,10 +34,107 @@
 
 import pandas.io.formats.printing as printing
 
+# A token value Python's tokenizer probably will never use.
```

Comment:
rather than do this here, pls make another module, maybe pandas/core/compuatation/tokenize.py

Reply:
`tokenize` is already a Python module. Should we call it `query_parsing`?

Reply:
maybe call it parsing